core/src/message/reply.rs

```diff
@@ -26,7 +26,9 @@ use crate::{
 use codec::{Decode, Encode};
 use scale_info::TypeInfo;
 
-/// Reply message.
+/// Message for Reply entry point.
+/// Contains result of exectuted program as exit code.
```

Comment:
result of the program?

Reply:
Main feature of the replies is storing message id replied on. It helps us realize chains of messages, async logic etc. There may not be more than 1 reply produced from message execution, etc.

Reply:
Rewrote it.